include/evmc/evmc.h

```diff
@@ -1,141 +1,177 @@
-/// EVMC -- Ethereum Client-VM Connector API
-///
-/// ## High level design rules
-///
-/// 1. Pass function arguments and results by value.
-///    This rule comes from modern C++ and tries to avoid costly alias analysis
-///    needed for optimization. As the result we have a lots of complex structs
-///    and unions. And variable sized arrays of bytes cannot be passed by copy.
-/// 2. The EVM operates on integers so it prefers values to be host-endian.
-///    On the other hand, LLVM can generate good code for byte swaping.
-///    The interface also tries to match host application "natural" endianess.
-///    I would like to know what endianess you use and where.
-///
-/// ## Terms
-///
-/// 1. EVM  -- an Ethereum Virtual Machine instance/implementation.
-/// 2. Host -- an entity controlling the EVM. The Host requests code execution
-///            and responses to EVM queries by callback functions.
-///
-/// @defgroup EVMC EVMC
-/// @{
+/**
+ * EVMC: Ethereum Client-VM Connector API
+ *
+ * @copyright
+ * Copyright 2018 Pawel Bylica.
```

Comment:
Btw, if there's a copyright here then I guess all the authors should be listed.

Probably way better is to have an `AUTHORS` file to list everyone, while the header can list the major contributors.

Reply:
I share the same opinion, but getting a clarification would be nice. See https://github.com/ethereum/evmc/pull/1.